Beyond06-SOW.md

```diff
@@ -0,0 +1,346 @@
+|                |                        | Incorporate presenting of the 1st exercise and the skill evaluation form in the 1st lecture                                                                                                                                                                  | Barak Korren         |
+| -              | 15.10                  | First challenge   self evaluation form submission date                                                                                                                                                                                                       | (Self work)          |
+| 2              | 31.10                  | Team collaboration: Git, GitHub, PRs and issues.                                                                                                                                                                                                             | Liora Milbaum        |
+|                |                        | Practicing Git (challenges)                                                                                                                                                                                                                                  | Liora Milbaum        |
+|                |                        | Incorporate presentation of the Git pair challenge into the 2nd lecture                                                                                                                                                                                      | Barak Koren          |
+| -              |                   | Self learning: Using Git in a team challenge And division into project groups                                                                                                                                                                                | (Self work in pairs) |
+| -              |                  | Git challenge submission date                                                                                                                                                                                                                                | (Self work)          |
+| 3              |                  | Advanced Git and GitHub skills                                                                                                                                                                                                                               | Liora Mibaum         |
+| 4              | 07.11                  | Demo 1:  Pre-requirements  fully working env (PRs, vagrant, issues with dependencies) Student 1: vagrant env Student 2: Django Student 3: Home page Student 4: CI with flake8 Student 5: ReadMe page (in git)  Demo: Guided Demo: Project presentation, MVP  |                      |
```

Comment:
No need to write the full requirements in this table, they are specified above already